lib/sucker_punch/queue.rb

```diff
@@ -39,7 +39,8 @@ def registered?
     end
 
     def name
-      klass.to_s.underscore.to_sym
+      klass_name = klass.to_s.underscore
+      "#{SuckerPunch::NAME_PREFIX}_#{klass_name}".to_sym
```

Comment:
I think this will be fine. I'd like to use `SuckerPunch::Queue` as the way to get the queue, so I don't think it matters for the user to know anything about how they're registered in the Celluloid registry.